integration-tests/testconfig/ocr/ocr.toml

```diff
@@ -171,4 +171,26 @@ selected_networks=["Anvil"]
 enabled = true
 next_gas_limit_percentage = 0.5
 delay_create = "1m"
-duration = "3m"
\ No newline at end of file
+duration = "3m"
+
+[TestOCRSoak_RPCDownForAllCLNodes.Common]
+chainlink_node_funding = 100
+[TestOCRSoak_RPCDownForAllCLNodes.OCR.Common]
+test_duration="15m"
+[TestOCRSoak_RPCDownForAllCLNodes.OCR.Soak]
+ocr_version="1"
+number_of_contracts=2
+time_between_rounds="1m"
+[TestOCRSoak_RPCDownForAllCLNodes.Network]
+selected_networks=["simulated"]
+
+[TestOCRSoak_RPCDownForHalfCLNodes.Common]
+chainlink_node_funding = 100
```

Comment:
this is excessive, 0.5 is enough on Sepolia for a 24h test :-)

Reply:
@Tofel I pushed a new commit with the fix [Update chainlink_node_funding](https://github.com/smartcontractkit/chainlink/pull/13624/commits/45709f52f4d2b738be7f263ad0602723b74ad07a)